pkgs/linux-cachyos/config.nix

```
@@ -0,0 +1,89 @@
+{ lib, ... }:
+with lib.kernel; {
+  LOCALVERSION = freeform "-cachyos-bore";
+  EXPERT = yes;
+  WERROR = no;
+
+  # Bore scheduler
+  SCHED_BORE = yes;
+
+  # Tick to 750hz
+  HZ = freeform "500";
+  HZ_500 = yes;
+
+  # Disable MQ Deadline I/O scheduler
+  MQ_IOSCHED_DEADLINE = lib.mkForce no;
+
+  # Disable Kyber I/O scheduler
+  MQ_IOSCHED_KYBER = lib.mkForce no;
+
+  # Enabling full ticks
+  CONTEXT_TRACKING = yes;
+  CONTEXT_TRACKING_FORCE = option no;
+  HZ_PERIODIC = no;
+  NO_HZ = yes;
+  NO_HZ_COMMON = yes;
+  NO_HZ_FULL = yes;
+  NO_HZ_FULL_NODEF = option yes;
+  NO_HZ_IDLE = no;
```

The others are the upstream value.
```suggestion
  HZ_PERIODIC = no;
  NO_HZ_FULL_NODEF = option yes;
  NO_HZ_IDLE = no;
```